packages/components/src/List/List.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import classnames from "classnames";
+import { camelCase, get, groupBy } from "lodash";
+import styles from "./List.css";
+import sectionStyles from "./SectionHeader.css";
+import { ListItem, ListItemProps } from "./ListItem";
+import { Typography } from "../Typography";
+
+interface ListProps {
+  /**
+   * Array of the list items.
+   *
+   * {@link https://atlantis.frend.space/components/list#list-item-props List Item Props}
+   */
+  readonly items: ListItemProps[];
+
+  /**
+   * Automatically colors the section header to match Jobbers system if a
+   * section is present.
+   *
+   * @default true
+   */
+  readonly jobberSectionColors?: boolean;
```

Comment:
This would be removed along with this comment https://github.com/GetJobber/atlantis/pull/152/files#r340812758.